vhost-device-scmi/src/devices/iio.rs

```diff
@@ -62,6 +62,11 @@ const UNIT_MAPPING: &[UnitMapping] = &[
         unit: scmi::SENSOR_UNIT_RADIANS_PER_SECOND,
         unit_exponent: 0,
     },
+    UnitMapping {
+        channel: "in_anglvel",
```

Comment:
Shouldn't this replace "in_anglevel" above? It seems "in_anglevel" doesn't exist, it's probably a typo and should be "in_anglvel" actually.

Reply:
Oh, you are right, I checkd variable "_iio_chan_type_name_spec_" in kernel "drivers/iio/industrialio-core.c", for iio device, it actually no "**_in_anglevel_**".
Replace previours definiation is good suggestion.
Maybe other channel name should also be checked. (For example, "**_in_angle_**" -> "**_in_angl_**")